backpack-lint/src/main/java/net/skyscanner/backpack/lint/check/BpkComposeComponentUsageDetector.kt

```diff
@@ -83,6 +83,14 @@ class BpkComposeComponentUsageDetector : Detector(), SourceCodeScanner {
                 "androidx.compose.foundation.text.BasicTextField",
             ),
         ),
+        PROGRESS_BAR(
```

Comment:
as we no longer expose material we only define this if there's non-material alternatives

Reply:
Ahh, i did wonder. Done